src/main/java/com/smartcar/sdk/data/VehicleChargingPort.java

```diff
@@ -0,0 +1,32 @@
+package com.smartcar.sdk.data;
+
+public class VehicleChargingPort {
```

Comment:
question: does this class not need to extend `ApiData`?

Reply:
Good catch!!. It needs the ApiData